platform/api/gcloud/compute.go

```diff
@@ -66,7 +73,7 @@ func (a *API) mkinstance(userdata, name string, keys []*agent.Key) *compute.Inst
 				Type:       "PERSISTENT",
 				InitializeParams: &compute.AttachedDiskInitializeParams{
 					DiskName:    name,
-					SourceImage: prefix + "/global/images/" + a.options.Image,
+					SourceImage: imagePrefix + "/global/images/" + a.options.Image,
```

Comment:
If Image is the full path `https://www.googleapis.com/compute/v1/projects/projname/global/images/imgname` Won't this create URLs like `https://www.googleapis.com/compute/v1/projects/projname/global/images/imgname/global/images/https://www.googleapis.com/compute/v1/projects/projname/global/images/imgname`?